mashup.js

```diff
@@ -0,0 +1,144 @@
+// Leave this in so, I can cut and past the URL to the browse
+// http://content.guardianapis.com/search?q=Mammal&order-by=relevance&page-size=50&show-fields=thumbnail%2Cbyline&show-blocks=body&api-key=2700505f-b667-4f0a-adc0-b024f02abe57
+
+var Url = function(title) {
+    this.endpoint = 'http://content.guardianapis.com/search';
+    this.query = 'q=' + title;
+    this.orderBy = 'order-by=relevance';
+    this.pageSize = 'page-size=50';
+    this.showFields = 'show-fields=' + escape('thumbnail,byline');
+    this.showBlocks = 'show-blocks=body';
+    this.apiKey = 'api-key=2700505f-b667-4f0a-adc0-b024f02abe57';
+};
+
+Url.prototype.fullUrl = function() {
+    var url = this.endpoint + '?' + this.query + '&' + this.orderBy + '&' + this.pageSize;
+    url += '&' + this.showFields + '&' + this.showBlocks + '&' + this.apiKey;
```

Comment:
FYI that [`$.param()`](http://api.jquery.com/jquery.param/) would be useful here.